src/main/java/lesson8Task4/Person.java

```diff
@@ -0,0 +1,52 @@
+package lesson8Task4;
+
+import java.util.List;
+
+public class Person {
```

Comment:
You have to implement Comparable interface to fulfill one of task conditions about default sorting